models/topics-models.js

```diff
@@ -45,4 +45,16 @@ exports.postAComment = (article_id, body) => {
     .then((comment) => {
         return comment.rows[0];
     });
-}
\ No newline at end of file
+}
+
+exports.updateArticleByID = (article_id, body) => {
+    return db.query(`SELECT votes FROM articles WHERE article_id = '${article_id}';`)
```

Comment:
We could achieve the same result without this query and wanted votes as we can set votes to votes+ body.inc_votes 🙂